AndroidSDKTests/src/test/java/com/leanplum/_whitebox/utilities/ResponseHelper.java

```diff
@@ -74,4 +74,19 @@ public static void seedResponse(String filename) {
       Log.e("ResponseHelper", "Unable to seed response from file: " + filename);
     }
   }
+
+  /**
+   * Seeds the Null response to Util.getResponse method.
+   *
+   */
+
+
+  public static void seedResponseNull() {
+    try {
+      doReturn(false).when(Util.class, Util.isConnected());
+      //doReturn(parseResponse(filename)).when(Util.class, "getResponse", anyObject());
```

Comment:
please could we just remove this completely, instead of commenting out?

Reply:
Nice catch. done. my bad actually.